packages/build-tools/cmd/posix_helpers.go

```diff
@@ -114,18 +114,18 @@ var rmCmd = &cobra.Command{
 
 		for _, item := range items {
 			info, err := os.Stat(item)
-			if err != nil && !force {
-				return eris.Wrapf(err, "Could not stat %s", item)
-			}
-
-			if info.IsDir() && !recursive {
+			if err != nil {
+				if !(force && eris.Is(err, os.ErrNotExist)) {
```

Comment:
I think `!force || !eris.Is(err, os.ErrNotExist)` would be easier to read. Can you change this?

Reply:
Yeah that's probably a bit more understandable.